docs/modules/record-encryption/aws-kms/con-aws-kms-setup.adoc

```diff
@@ -12,17 +12,17 @@ filter to be kept separate from any keys used by other systems. This document as
 will be to prefix the alias used by the filter with the word `KEK_`. If a different naming convention is used, adapt
 the instructions accordingly.
 
-== Administrator Actor
+== Role of the Administrator
 
-To use the filter, there must be an administrative identity established within AWS IAM.  This user, which is likely to be a human,
-has the responsibility to manage keys and aliases within AWS KMS for use by the filter.
+In order to use the filter, there must be an external entity which is responsible for creating encryption keys within AWS KMS which will be used to encrypt the records. It is up to the organization
+deploying Record Encryption to decide how this entity is managed.
 
-Use AWS IAM to create a `kroxylicious-admin` user. Attach the policies `AWSKeyManagementServicePowerUser` and `IAMUserChangePassword`
-to the user. You may wish to attach policy `AWSCloudShellFullAccess` so the Administrator can use the AWS CloudShell to
-use CLI to manage the KEKs. Grant the user access to the Console.
+The *Administrator* must have permissions to create keys in AWS KMS.  As a starting point, the
+built-in AWS policy `AWSKeyManagementServicePowerUser` confers sufficient key management privileges.
 
-If using the CLI, the following commands can be used to establish the Administrator user.  This example illustrates using the
-user-name `kroxylicious-admin`.  The choice of name is not significant.  If a different user-name is used, adapt the
+To help get you started, the following commands can be used to establish an Administrator user with permissions
+suitable for managing encryption keys in KMS via an AWS Cloud Shell. This example illustrates using the user-name
+`kroxylicious-admin` but the choice of name is not significant.  If a different user-name is used, adapt the
```

Comment:
Didn't we agreed to add a comment here for testing purposes to fulfill this issue #1348 ?? For those cases where a deletion of the key is needed.

I think adding a link to the AWS developer docs would be sufficient.

Reply:
On #1348 i commented

> I don't think there is a case for documenting the permissions required for deleting keys until there exists a procedure to work out if a key is not in use.

so I think we should not add the link right now.